.github/workflows/test-azureblob.yml

```diff
@@ -0,0 +1,46 @@
+on:
+    push:
+        branches:
+            - master
+            - main
+    pull_request:
+name: "Tests Azure Blob"
+jobs:
+    Tests:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                go-version:
+                    - 1.18.x

```

Comment:
Add 1.16.x here

Reply:
It will not work with go 1.16. Minimum required version is 1.18. [Source](https://github.com/Azure/azure-sdk-for-go/blob/main/README.md)

Reply:
https://i.imgur.com/T69oUmA.png

Reply:
@efectn @ReneWerner87 Thoughts on this? My understanding is that Fiber supports 1.16.

Reply:
It's OK to me since it supports 1.18 as minimum version but we can add 1.19

Reply:
@efectn 1.19 already added.